lib/config/options/index.ts

```diff
@@ -852,7 +852,7 @@ const options: RenovateOptions[] = [
   {
     name: 'matchBaseBranches',
     description:
-      'List of branches to match (e.g. ["master"]). Valid only within `packageRules` object.',
+      'List of strings containing exact matches (e.g. `["master"]`) and/or regex expressions (e.g. `["/^release\\/.*/"]`). Valid only within `packageRules` object.',
```

Comment:
```suggestion
      'List of strings containing exact matches (e.g. `["main"]`) and/or regex expressions (e.g. `["/^release\\/.*/"]`). Valid only within `packageRules` object.',
```

I'd rather use `main` here, now that many Git hosting providers have switched to this name for the default branch. 😉

Reply:
Of course, stupid of me. I'm so used to master yet. I changed it

Reply:
Thank you for applying the suggestion! 😄 

Don't feel bad, the reference to `master` was in there before you touched the file, I should have updated this sooner. 🙈